agent/src/runtime_connectors/podman_cli.rs

```diff
@@ -91,9 +105,15 @@ impl TimedPodmanPsResult {
     }
 
     async fn new_inner() -> PodmanPsCache {
+        let mut res = PodmanCli::list_states_internal().await;
+        if res.is_err() {
```

Comment:
Should we check what kind of Podman error this is and act only on the DB problem?

Reply:
Most likely we could do that, but I do not know if this issue is caused only by the internal DB problem of the Podman. The second thing is that such thing would not be simple to implement. 
Our helper function does not return the exit code. It would mean to change the helper function. Bigger problem is that the `podman ps` does not have documented "exit codes". For us it would be rather guessing which exit code means a problem with DB. To be on the safe side I would suggest to do the retry after each error.

Reply:
Alright 👍